docs/codeql/codeql-cli/about-codeql-packs.rst

```diff
@@ -73,7 +73,7 @@ The following properties are supported in ``qlpack.yml`` files.
    * - ``suites``
      - ``octo-org-query-suites``
      - Optional
-     - The path to a directory that contains the "well-known" query suites in the pack, defined relative to the pack directory. You can run "well-known" suites stored in this directory by specifying the pack name, without providing their full path. For more information about query suites, see ":doc:`Creating CodeQL query suites <creating-codeql-query-suites>`."
+     - The path to a directory in the pack that contains the query suites you want to make known to the CLI, defined relative to the pack directory. QL pack users can run "well-known" suites stored in this directory by specifying the pack name, without providing their full path. This is not supported for CodeQL packs downloaded from a package registry. For more information about query suites, see ":doc:`Creating CodeQL query suites <creating-codeql-query-suites>`."
```

Comment:
This looks good. The behaviour is complicated and not intuitive, but I think you captured the essence of it.